app/models/conference/importer.rb

```diff
@@ -57,17 +63,12 @@ def process_csv(file)
             url: row['Website'],
             notes: row['Notes'],
           }
-          conference.update!(conference_hash.merge(season_id: fetch_season_id(row['UID'])))
+          conference.update!(conference_hash)
```

Comment:
You can merge the `season_id` from the line below onto `conference_hash`, saving one DB write.